tests/jest/BookNavigator/search/search-results.test.js

```diff
@@ -81,21 +81,21 @@ describe('<ia-book-search-results>', () => {
     sinon.replace(IABookSearchResults.prototype, 'createRenderRoot', function createRenderRoot() { return this; });
     const el = await fixture(container(results));
 
-    expect(el.innerHTML).toContain(`${results[0].title}`);
+    expect(el.innerHTML).toInclude(`${results[0].title}`);
```

Comment:
These should be `toContain`; jest doesn't have a `.toInclude` method, so that needs to be changed over.

Reply:
thanks, i fixed it